third_party/3/orjson.pyi

```diff
@@ -0,0 +1,9 @@
+# https://github.com/ijl/orjson/blob/master/orjson.pyi
+
+from typing import Any, Callable, Optional, Union
+
+def dumps(obj: Any, default=Optional[Callable[Any]]) -> bytes: ...
```

Comment:
I think you mixed up the default and the type here.

Reply:
Fixed, I think, thanks.